gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/JWTMessages.java

```diff
@@ -41,6 +41,9 @@
   @Message( level = MessageLevel.INFO, text = "Unable to verify token: {0}" )
   void unableToVerifyToken(@StackTrace( level = MessageLevel.ERROR) Exception e);
 
+  @Message( level = MessageLevel.ERROR, text = "Unable to verify token expiration: {0}" )
```

Comment:
Do we need this at an error level?

Reply:
I went back and forth with this, and could go either way really. Do you think INFO is sufficient?

Reply:
INFO or WARN should be good I think. I just saw the message above about unable to verify being INFO and was like hmmm that seems worse than not being able to verify the expiration.